src/server/pps/server/githook/githook.go

```diff
@@ -121,11 +128,7 @@ func (s *gitHookServer) findMatchingPipelineInputs(payload github.PushPayload) (
 	return pipelines, inputs, nil
 }
 
-func (s *gitHookServer) HandlePush(payload interface{}, _ webhooks.Header) (retErr error) {
-	pl, ok := payload.(github.PushPayload)
-	if !ok {
-		return fmt.Errorf("received invalid github.PushPayload")
-	}
+func (s *gitHookServer) HandlePush(pl github.PushPayload) (retErr error) {
```

Comment:
nit: It'd be nice to have a comment here on what `HandlePush` is for at a high level. My understanding is for pipelines with input type of `git` when pushes are made to the relevant branch, we commit the payload to the corresponding pfs repo.

Reply:
I'm actually not sure why this is publicly exported. It's not used outside of this struct. I'll swap it over. Would you like me to still add a comment?

Reply:
Ah, I see. In that case, I think a comment would make more sense at the package declaration level.